pkg/loop/config.go

```diff
@@ -6,10 +6,18 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"time"
 )
 
 const (
-	envDatabaseURL            = "CL_DATABASE_URL"
+	envDatabaseURL                           = "CL_DATABASE_URL"
+	envDatabaseDefaultIdleInTxSessionTimeout = "CL_DATABASE_DEFAULT_IDLE_IN_TX_SESSION_TIMEOUT"
+	envDatabaseDefaultLockTimeout            = "CL_DATABASE_DEFAULT_LOCK_TIMEOUT"
+	envDatabaseDefaultQueryTimeout           = "CL_DATABASE_DEFAULT_QUERY_TIMEOUT"
```

Comment:
Does `DEFAULT` add value here?
```suggestion
	envDatabaseIdleInTxSessionTimeout = "CL_DATABASE_IDLE_IN_TX_SESSION_TIMEOUT"
	envDatabaseLockTimeout            = "CL_DATABASE_LOCK_TIMEOUT"
	envDatabaseQueryTimeout           = "CL_DATABASE_QUERY_TIMEOUT"
```

Reply:
have these been released? will we break someone if they are changed? if not, then yes rename them

Reply:
yeah, default does not add anything